packages/cli/src/commands/graph/tasks/graphOrderTask.ts

```diff
@@ -71,51 +89,15 @@ export function graphOrderTask(
         });
       }
 
-      // set the order on the context so we can use it in other tasks
-      ctx.migrationOrder = order;
-
-      // skip the output and prompt. for when we just want the order
-      if (options.skipPrompt) {
-        ctx.sourceFilesRel = order.packages.flatMap((pkg) => pkg.files);
-        ctx.sourceFilesAbs = ctx.sourceFilesRel.map((file) => resolve(rootPath, file));
+      if (!output && !skipPrompt) {
```

Comment:
`!skipPrompt` now that the prompt is removed we can remove this conditional and wherever its being set